mungegithub/submit-queue/www/script.js

```diff
@@ -235,15 +238,24 @@ function SQCntl(dataService, $interval, $location) {
           failedBuild = true;
       }
       obj.stability = '';
-      result.push(obj);
+      if (!obj.msg) {
+        result.push(obj);
+      } else {
+        if (obj.msg.includes("[nonblocking]")) {
```

Comment:
can/should we remove [nonblocking] from the obj.msg here? It's just a distraction at this point.